package.json

```diff
@@ -3,7 +3,6 @@
   "version": "1.16.24",
   "description": "Next generation Brave browser for macOS, Windows, Linux, and eventually Android",
   "scripts": {
-    "audit_deps": "npm --prefix src/brave run audit_deps --",
```

Comment:
@mihaiplesa i removed this because it's run as part of the test-security script, but lmk if it's needed because it's called directly somewhere.